cloudformation/imagebuilder/pcluster_install.yaml

```diff
@@ -133,33 +174,35 @@ phases:
                 apt-get -y install build-essential curl wget jq
               fi
 
+      # Download and vendor Cookbook
       - name: DownloadCookbook
         action: ExecuteBash
         inputs:
           commands:
             - |
               set -x
-              CookbookVersion="aws-parallelcluster-cookbook-2.10.1"
-              cookbook_url=https://${AWS::Region}-aws-parallelcluster.s3.${AWS::Region}.amazonaws.com/cookbooks/${!CookbookVersion}.tgz
-              echo "${!cookbook_url}"
+              # TODO move vars out in separate actions
```

Comment:
Do you want to address the TODO in this patch?

Reply:
done